chain/ethereum/src/ethereum_adapter.rs

```diff
@@ -1785,7 +1785,21 @@ async fn fetch_transaction_receipts_in_batch_with_retry(
             fetch_transaction_receipts_in_batch(web3, hashes, block_hash, logger).boxed()
         })
         .await
-        .map_err(|_timeout| anyhow!(block_hash).into())
+        .map_err(|timeout| {
```

Comment:
Oups leftover from understanding an error I had. Will open a dedicated PR for those.